packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/PayoutCard.tsx

```diff
@@ -41,16 +57,7 @@ export const PayoutCard = ({
             <Icon icon="CALENDAR_THIN" color={theme.iconSubdued} />
 
             <CardBottomContent>
-                <AccentP>
-                    {payout === undefined ? (
-                        <Translation
-                            id="TR_STAKE_MAX_REWARD_DAYS"
-                            values={{ days: BACKUP_REWARD_PAYOUT_DAYS }}
-                        />
```

Comment:
I think something like this should work
```
<Translation
  id="TR_STAKE_DAYS"
  values={{ count: payout }}
/>
```

Reply:
https://github.com/trezor/trezor-suite/pull/12553/commits/a8302aae456b37cedd07ecff8b47a27a20c49e9f